src/server/migrations/20200602194903_users.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('users', (table) => {
+    table.increments();
```

Comment:
Shouldn't be this `table.increments('id')`

Reply:
You cab write it there, and you get the same result, so with or without (id) its the same